### PR TITLE
added profile

### PR DESCRIPTION
**Pull request by nastroh22**

### CODE REVIEW

**Comment by nastroh22** [comment]
I tried the build locally too but lmk if there any issues

File: _people/nick_strohmeyer.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Nick Strohmeyer
+position: gradstudent
+avatar: nick_strohmeyer.png
+---
+
+
+<img width="300" src="{{site.baseurl}}/images/people/{{page.avatar}}" data-action="zoom">
+
+- _Master's student, Electrical & Computer Engineering, UT Austin_<br>
```
Comment (dfridovi) [inline _people/nick_strohmeyer.md:10]: You are a PhD student, right?

File: _people/nick_strohmeyer.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+<i class="fa fa-envelope-o"></i> `nstrohmeyer@utexas.edu`<br>
+<i class="fa fa-newspaper-o"></i> [CV (pdf)](/documents/nick_strohmeyer_cv.pdf)
+
+**Office**<br>
+Anna Hiss Gym <br>
+2501 Wichita St,
+Austin, TX 78712
+
+[Nick Strohmeyer](https://www.linkedin.com/in/nick-strohmeyer-209a3a157/) is a Master's student co-advised by Sriram Vishwanath in the [WINCG Group](https://www.ece.utexas.edu/research/groups/wireless-networking-and-communications-group-wncg). His research interests lie in the use of optimal control, machine learning, and game theory for modeling dynamical systems and the design of behaviorial planning algorithms. Prior to attending UT Austin, Nick worked as a Data Analyst in digital marketing and telcommunications for 3 years. He graduated with a B.S. in Mathematics from the University of San Francisco. In his free time, Nick enjoys playing recreational sports such as ice hockey, softball, and tennis and getting outdoors to hike or explore.
```
Comment (dfridovi) [inline _people/nick_strohmeyer.md:20]: Aren't you coadvised? I am very confused. I thought I was your coadvisor. What is going on?

Comment (nastroh22) [reply _people/nick_strohmeyer.md:20]: Yes, here I was just following others who listed their co-advisors as well

Comment (dfridovi) [reply _people/nick_strohmeyer.md:20]: Sorry I cannot read. Didn't see the "co" in "co-advised"

File: _people/nick_strohmeyer.md (hunk 1 above)
Comment (nastroh22) [reply _people/nick_strohmeyer.md:10]: Yes I'll change that

**Review by dfridovi — APPROVED** [review]

**Pull request merged**